powerauth-push-server/src/main/java/io/getlime/push/repository/InboxRepository.java

```diff
@@ -34,7 +35,7 @@
  * @author Petr Dvorak, petr@wultra.com
  */
 @Repository
-public interface InboxRepository extends PagingAndSortingRepository<InboxMessageEntity, String> {
+public interface InboxRepository extends PagingAndSortingRepository<InboxMessageEntity, String>, CrudRepository<InboxMessageEntity, String> {
```

Comment:
What a surprise! I did not find much explanation why `PagingAndSortingRepository` doest not extend `CrudRepository` anymore. https://github.com/spring-projects/spring-data-commons/issues/2537